injective/assetlist.json

```diff
@@ -1068,6 +1068,31 @@
       "type_asset": "cw20",
       "address": "inj19vy83ne9tzta2yqynj8yg7dq9ghca6yqn9hyej",
       "base": "cw20:inj19vy83ne9tzta2yqynj8yg7dq9ghca6yqn9hyej",
+      "name": "cw20 DRUGS",
+      "display": "DRUGS",
+      "symbol": "DRUGS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/cw20.drugs.png"
```

Comment:
I'll request that the (.) dot not be used in the file name, because it looks like drugs is the extension, rather than part of a given name.